src/products/components/ProductListDatagrid/utils.ts

```diff
@@ -0,0 +1,45 @@
+import { ProductFilterKeys } from "../ProductListPage";
+
+export const getPriceClickSearchParams = (search: string): string => {
+  const params = new URLSearchParams(search);
+
+  const filtersIndices = Array.from(params.keys())
+    .map(key => {
```

Comment:
this whole map can be a dedicated function, the name will indicate what you map to